src/main/resources/hibernate.cfg.xml

```diff
@@ -0,0 +1,15 @@
+<hibernate-configuration xmlns="http://www.hibernate.org/xsd/orm/cfg">
+    <session-factory>
+        <property name="connection.url">jdbc:mysql://localhost:3306/many_to_many?serverTimeZone=UTC</property>
+        <property name="connection.driver_class">com.mysql.cj.jdbc.Driver</property>
+        <property name="connection.username">root</property>
+        <property name="connection.password">fkpQ88wV9Aiqh-$</property>
+        <property name="hibernate.show_sql">true</property>
+        <property name="hbm2ddl.auto">create-drop</property>
+
+        <mapping class="mate.academy.hibernate.relations.model.Actor"/>
+        <mapping class="mate.academy.hibernate.relations.model.Country"/>
+        <mapping class="mate.academy.hibernate.relations.model.Movie"/>
+
+    </session-factory>
+</hibernate-configuration>
```

Comment:
```suggestion
</hibernate-configuration>

```

Reply:
not fixed, each file should end with exactly one empty line
![image](https://github.com/mate-academy/hibernate-relations-hw/assets/49767049/d08c82b3-c296-4be1-a5b2-79b9bc802e50)